lib/logger-helper.js

```diff
@@ -57,98 +60,100 @@ var auditResponse = function(req, res, options){
             auditObject = Object.assign(auditObject, req.additionalAudit);
         }
 
-        options.logger.info(auditObject);
+        options.logger.info(auditObject, 'Inbound Transaction');
     }
 };
 
 function getRequestAudit(req, options){
-    var headers = req && req.headers ? req.headers : NA;
+    var headers = _.get(req, 'headers');
     var requestFullURL = utils.getUrl(req);
     var requestRoute = utils.getRoute(req);
     var queryParams = req && req.query !== {} ? req.query : NA;
     var method = req && req.method ? req.method : NA;
     var URLParams = req && req.params ? req.params : NA;
     var timestamp = req && req.timestamp ? req.timestamp.toISOString() : NA;
     var timestamp_ms = req && req.timestamp ? req.timestamp.valueOf() : NA;
+    var requestBody =  _.get(req, 'body');  //handle body clone the original body
 
-    var requestBody;
+    requestBody = handleJson(requestBody, options.logger, options.request.excludeBody, options.request.maskBody);
```

Comment:
Why not calling it like that?

`requestBody = handleJson(requestBody, options);`

Reply:
because I call it in the responseAudit as well - excludebody and maskBody are under options.response this time

Reply:
and when you call handleJson for headers or body the field from options are diff:
1. headers -  excludHeaders and maskHeaders
2. body - excludBody and maskBody